src/styles/styles.js

```diff
@@ -2171,16 +2171,18 @@ const styles = {
     }, 0),
 
     iouPreviewBox: {
-        backgroundColor: themeColors.componentBG,
-        borderColor: themeColors.border,
-        borderWidth: 1,
+        backgroundColor: themeColors.cardBG,
         borderRadius: variables.componentBorderRadiusCard,
         padding: 20,
         marginTop: 16,
         maxWidth: variables.sideBarWidth,
         width: '100%',
     },
 
+    iouPreviewBoxHover: {
+        backgroundColor: themeColors.border,
```

Comment:
This is a wrong variable. Either create a new one or use some other variable. Using wrong theme variable makes it harder to customize the theme.

Reply:
Good point, I found a better descriptive color. Just a heads up, we don't want to add any new theme variables until we switch to the new color mode

Reply:
Got it.